src/android/com/adobe/phonegap/push/BackgroundActionButtonHandler.java

```diff
@@ -9,7 +9,7 @@
 import android.support.v4.app.RemoteInput;
 
 public class BackgroundActionButtonHandler extends BroadcastReceiver implements PushConstants {
-    private static String LOG_TAG = "PushPlugin_BackgroundActionButtonHandler";
+    private static String LOG_TAG = "ActionButtonHandler";
```

Comment:
I'd prefer to keep `PushPlugin_` at the start of the tag so I can filter all the logs coming from the push plugin easier so how about something like `PushPlugin_BGActionBtn`?